mLRS_Flasher.py

```diff
@@ -92,7 +92,10 @@ def flashSTM32CubeProgrammer(programmer, firmware):
     if sys.platform.lower() == 'darwin':
         ST_Programmer = os.path.join('thirdparty','STM32CubeProgrammer','mac','bin','STM32_Programmer_CLI')
     else:
-        ST_Programmer = os.path.join('thirdparty','STM32CubeProgrammer','win','bin','STM32_Programmer_CLI.exe')
+        if sys.platform.lower() == 'linux':
```

Comment:
shouldn't this be an elif ... insetad of else: if ...?

Reply:
Sure.